src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -10,13 +10,7 @@
 import seedu.address.commons.core.index.Index;
 import seedu.address.commons.util.StringUtil;
 import seedu.address.logic.parser.exceptions.ParseException;
-import seedu.address.model.person.Address;
-import seedu.address.model.person.Email;
-import seedu.address.model.person.Major;
-import seedu.address.model.person.Meeting;
-import seedu.address.model.person.Name;
-import seedu.address.model.person.Phone;
-import seedu.address.model.person.Role;
+import seedu.address.model.person.*;
```

Comment:
I understand using `*` makes it a lot easier for now, but we should remember to change it before product release and to pass checkstyle.